python/cuml/common/input_utils.py

```diff
@@ -78,10 +77,11 @@ def input_to_cuml_array(X, order='F', deepcopy=False,
     X : cuDF.DataFrame, cuDF.Series, numba array, NumPy array or any
         cuda_array_interface compliant array like CuPy or pytorch.
 
-    order: string (default: 'F')
-        Whether to return a F-major or C-major array. Used to check the order
-        of the input. If fail_on_order=True method will raise ValueError,
-        otherwise it will convert X to be of order `order`.
+    order: 'F', 'C' or 'K' (default: 'F')
+        Whether to return a F-major ('F'),  C-major ('C') array or Keep ('K')
```

Comment:
Nitpick: Should we (eventually) default to keep so that algorithms have to be explicit about their expected order?  I’m just thinking it might make increase readability a bit, while the expectation of not specifying this value would mean no change.

Reply:
yeah, I think that's a good idea, will create a github issue. It definitely will improve readability for new devs as well as being more clear.